docs/testing-styleguide.md

```diff
@@ -229,50 +221,4 @@ Other support files include the `helper.js` file which contains functions such a
 
 
 ##### Test Results
-Depending on how Cypress was run, you will find test results in either the Cypress Test Runner UI, the console or in GitHub Actions linked from the build checks in GitHub. For further information see the [Getting Started Guide](../cypress/README.md).
-
-##### Applitools
-###### Integration with Cypress
-Instructions for adding Applitools to an existing Cypress project can be found [here](https://applitools.com/tutorials/cypress.html). Applitools is integrated with Cypress by means of various configuration and settings in the `cypress/support/index.js` file, `applitools.config.js` file, and with additional feature files and corresponding step definitions of those components/stories where we want to do visual testing. When the relevant tests are run from Cypress, snapshots will be taken where specified in the feature files. We currently run visual tests in Chrome browser only.
-
-An example of a feature file for a visual testing step follows. The `Then` step is where Applitools will take a snapshot:
-
-```
-Feature: Button component - primary type
-  I want to check that all examples of Button component - primary type render correctly per theme
-
-  @positive
-  @applitools
-  Scenario Outline: Check that primary buttons render correctly with theme set to <theme>
-    When I open "button" component primary_buttons story with theme "<theme>"
-    Then Element displays correctly
-    Examples:
-      | theme  |
-      | mint   |
-      | aegean |
-      | none   |
-```
-
-Results can be found in the [Applitools dashboard](https://eyes.applitools.com/app/test-results/00000251812838508713/?accountId=MZDiTwN5_kOmMbjBqRi9pw~~) for our open source Sage team account. It is a public account so all interested parties should be able to view the results providing they have access. If you do not have access and believe you need it, contact the Carbon Team.
-
-###### Passing or failing tests
-When tests have passed, i.e. there are no differences between the new snapshot and the baseline snapshot, the tests will be marked as “Passed” in Applitools. If the test has failed it will be marked as “Unresolved”. Differences between the baseline snapshot and the one from the current run are highlighted in pink:
-
-![Differences](./images/applitools_differences_identified.png)
```

Comment:
The images in `docs/images` contained in this section at lines 261 and 268 should be deleted from this location.